airflow/hooks/base_hook.py

```diff
@@ -67,7 +68,7 @@ def _get_connection_from_env(cls, conn_id):
         return conn
 
     @classmethod
-    def get_connections(cls, conn_id):
+    def get_connections(cls, conn_id):  # type: (str) -> Iterable[Connection]
```

Comment:
In my other projects, we use the following syntax:
```
def get_connections(cls, conn_id : str) -> Iterable[Connection]:
```

I wonder whether there is a preferred style of mypy syntax.

Reply:
Or is it because we need to support py27?

Reply:
Yep, we have to use this syntax as long as we're supporting python2: https://mypy.readthedocs.io/en/latest/python2.html. If we drop python2 support, we can switch to python3-style annotations.